lib/jgi_mg_assembly/pipeline_steps/spades.py

```diff
@@ -2,6 +2,7 @@
 Handles running SPAdes from command line, using the command and parameters provided
 by the JGI Metagenome Assembly Pipeline.
 """
+from step import Step
 import os
 import subprocess
```

Comment:
subprocess unused here

Reply:
I'm really not careful when copying and pasting...